applications/services/gui/canvas.c

```diff
@@ -137,6 +137,12 @@ void canvas_set_font(Canvas* canvas, Font font) {
     }
 }
 
+void canvas_set_custom_font(Canvas* canvas, const uint8_t* font) {
```

Comment:
Adding this will introduce a strict dependency on u8g2 as rendering backend.

Reply:
I don't know how to get around this, I can add intermediate function but it would call u8g2 anyway.

Reply:
All other functions are also dependent on u8g2 anyway... You can just replace the internal code and still use a pointer to whatever font data you want, just cast it to uint8_t if it is in other format when you call function and cast to whatever type the lib needs inside the function

Reply:
Maybe it would be better to pass some sort of font data type/backend id in that function, so the actual implementation could check if it can handle its data? @skotopes

Reply:
Hm, so I pass Canvas* pointer, let's sat void* pointer for font data, and some data type ID? May work... But the lib needs to support it

Reply:
I think this problem can be solved by simply renaming canvas_set_custom_font to canvas_set_custom_u8g2_font.

Reply:
rename is acceptable